### PR TITLE
12.0 Various snailmail Improvements

### PR DESCRIPTION
**Pull request by rrahir**
PRICING
--------------
The pricing on snailmail has changed and we now charge stamps for each letter.

 USABILITY
-----------------

Added a new error to help the user more specifically

Avoid spamming the user's mailbox with the next  activities when a letter can not be sent.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by rrahir** [comment]
@robodoo r+

**Comment by KangOl** [comment]
@robodoo r-

File: addons/snailmail_account/wizard/account_invoice_send.py [hunk 1]
```diff
@@ -11,17 +11,9 @@ class AccountInvoiceSend(models.TransientModel):
 
     partner_id = fields.Many2one('res.partner', compute='_get_partner', string='Partner')
     snailmail_is_letter = fields.Boolean('Send by Post', help='Allows to send the document by snail mail (coventional posting delivery service)', default=lambda self: self.env.user.company_id.invoice_is_snailmail)
-    snailmail_cost = fields.Float(string='Credits', compute='_snailmail_estimate', store=True, readonly=True)
-    currency_id = fields.Many2one('res.currency', compute='_get_invoice_currency', string="Currency")
```
Comment (KangOl) [inline addons/snailmail_account/wizard/account_invoice_send.py]: Don't remove field in stable version

File: addons/snailmail_account/wizard/account_invoice_send_views.xml [hunk 2]
```diff
@@ -9,14 +9,17 @@
...
                         <div name="option" class="text-left d-inline-block">
                             <field name="snailmail_is_letter" />
                             <b><label for="snailmail_is_letter"/></b>
                         </div>
                         <div class="text-left d-inline-block mr8" name='cost' attrs="{'invisible': [('snailmail_is_letter','=', False)]}">
                             <b>(
-                                <field name="snailmail_cost" class="mr4" widget='monetary' options="{'currency_field': 'currency_id'}"/>
+                                <span>
+                                    <field name="snailmail_cost" options="{'digits':[0,0]}" class="mr4"/>
+                                    <span attrs="{'invisible': [('snailmail_cost', '&gt;=', 2)]}">Stamp </span>
```
Comment (KangOl) [inline addons/snailmail_account/wizard/account_invoice_send_views.xml]: wont work well on other languages.
See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

Note that odoo does not currently support plurial form in translations

Comment (rrahir) [reply addons/snailmail_account/wizard/account_invoice_send_views.xml]: @KangOl SOrry for the field thing, I was backporting something we planned tor master and forgot about it. What would you suggest as a solution for the plural form ?

Comment (KangOl) [reply addons/snailmail_account/wizard/account_invoice_send_views.xml]: Only keeping plurial form should be ok.

File: addons/snailmail_account/wizard/account_invoice_send.py (hunk 1 above)
Comment (rrahir) [reply addons/snailmail_account/wizard/account_invoice_send.py]: restored

**Comment by KangOl** [comment]
@robodoo r+

**Pull request closed without merging**